qa/src/test/java/io/camunda/migrator/qa/CustomJobTypeValidationTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+package io.camunda.migrator.qa;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.migrator.RuntimeMigrator;
+import io.camunda.migrator.qa.util.RuntimeMigrationAbstractTest;
+import io.github.netmikey.logunit.api.LogCapturer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.test.context.TestPropertySource;
+
+@TestPropertySource(properties = {
+    "camunda.migrator.job-type=custom-job-type"
+})
+public class CustomJobTypeValidationTest extends RuntimeMigrationAbstractTest {
+
+  @RegisterExtension
+  protected LogCapturer logs = LogCapturer.create().captureForType(RuntimeMigrator.class);
+
+  @Test
+  public void shouldUseCustomJobTypeInValidationMessage() {
+    // given
+    deployProcessInC7AndC8("noMigratorListener.bpmn");
+
+    String id = runtimeService.startProcessInstanceByKey("noMigratorListener").getId();
+
+    // assume
+    assertThat(runtimeService.createProcessInstanceQuery().processInstanceId(id).singleResult()).isNotNull();
+
+    // when
+    runtimeMigrator.start();
+
+    // then
+    assertThatProcessInstanceCountIsEqualTo(0);
+
+    var events = logs.getEvents();
+    assertThat(events.stream()
+        .filter(event -> event.getMessage()
+            .matches(String.format(".*Skipping process instance with legacyId \\[%s\\]: "
+                + "Couldn't find execution listener of type 'custom-job-type' on start event "
+                + "\\[Event_1px2j50\\] in C8 process with key \\[(\\d+)\\]\\.", id))))
+        .hasSize(1);
```

Comment:
Nit
So `MigratorListenerNotFoundTest` covers the actual skipping logic and these tests are just there to ensure the error message is parsed correctly with custom configs right? In that case I think we could just use `logs.assertContains()` instead of this custom matching and asserting
(same suggestion also for the other test classes)

Reply:
Will try and am happy if this makes the assertions simpler.

Reply:
How can `logs.assertContains()` work with an expression like `\\[(\\d+)\\]\\`?

Reply:
Ah I see, if we have to use that type of matching then probably your way is better.